alkemyTech/OT102-Server
# [FIX] - model - migrations - middleware - errorhandler

NO TICKET

Description:
This PR address several fixes for models, migrations, seeder.
Adds a handler for errors and a factory for successful responses.

How to test:
start the server: `npm run start:dev`

then you can try some request to endpoints and see responses.
